lisa/features/infiniband.py

```diff
@@ -152,10 +152,10 @@ def _get_mofed_version(self) -> str:
         default = "5.4-3.0.3.0"
         if self._is_legacy_device():
             return "4.9-5.1.0.0"
-        if (
-            isinstance(node.os, Ubuntu) and node.os.information.version >= "20.4.0"
-        ) or (isinstance(node.os, Redhat) and node.os.information.version >= "8.2.0"):
+        if isinstance(node.os, Ubuntu) and node.os.information.version >= "20.4.0":
             return "5.7-1.0.2.0"
+        if isinstance(node.os, Redhat) and node.os.information.version >= "8.2.0":
```

Comment:
have you uploaded the ofed driver into our storage account?

Reply:
Yes.

Reply:
@contre I just double check, only MLNX_OFED_LINUX-5.8-1.1.2.1-rhel8.7-x86_64.tgz uploaded, that means, for other version, 8.6, 9.0, test case will fail for not find corresponding tar package

Reply:
I will get that taken care of tomorrow.